plugins/DeepJetTableProducer.cc

```diff
@@ -0,0 +1,315 @@
+      djTable->addColumn<float>("DeepJet_Npfcan_drminsv_" + s, Npfcan_drminsv_nNpf[p], "pseudoangular distance between the neutral candidate and the closest secondary vertex for the " + s + ". npf", nanoaod::FlatTable::FloatColumn, 10);
+      djTable->addColumn<float>("DeepJet_Npfcan_HadFrac_" + s, Npfcan_HadFrac_nNpf[p], "fraction of the neutral candidate energy deposited in the hadronic calorimeter for the " + s + ". npf", nanoaod::FlatTable::FloatColumn, 10);
+      djTable->addColumn<float>("DeepJet_Npfcan_ptrel_" + s, Npfcan_ptrel_nNpf[p], "fraction of the jet momentum carried by the neutral candidate for the " + s + ". npf", nanoaod::FlatTable::FloatColumn, 10);
+      djTable->addColumn<int>("DeepJet_Npfcan_isGamma_" + s, Npfcan_isGamma_nNpf[p], "integer flag indicating whether the neutral candidate is a photon for the " + s + ". npf", nanoaod::FlatTable::IntColumn, 10);
+      
+  }  
+
+  // ============================================================== SVs ===================================================================
+  for (unsigned int p = 0; p < n_sv_; p++) {
+      auto s = std::to_string(p);
+      
+      djTable->addColumn<float>("DeepJet_sv_mass_" + s, sv_mass_nSV[p], "SV mass of the " + s + ". SV", nanoaod::FlatTable::FloatColumn, 10);
+      djTable->addColumn<float>("DeepJet_sv_pt_" + s, sv_pt_nSV[p], "SV pt of the " + s + ". SV", nanoaod::FlatTable::FloatColumn, 10);
+      djTable->addColumn<float>("DeepJet_sv_ntracks_" + s, sv_ntracks_nSV[p], "Number of tracks asociated to the " + s + ". SV", nanoaod::FlatTable::FloatColumn, 10);
+      djTable->addColumn<float>("DeepJet_sv_chi2_" + s, sv_chi2_nSV[p], "chi2 of the " + s + ". SV", nanoaod::FlatTable::FloatColumn, 10);
+      djTable->addColumn<float>("DeepJet_sv_normchi2_" + s, sv_normchi2_nSV[p], "chi2/dof of the " + s + ". SV", nanoaod::FlatTable::FloatColumn, 10);
+      djTable->addColumn<float>("DeepJet_sv_dxy_" + s, sv_dxy_nSV[p], "2D impact parameter (flight distance) value of the " + s + ". SV", nanoaod::FlatTable::FloatColumn, 10);
+      djTable->addColumn<float>("DeepJet_sv_dxysig_" + s, sv_dxysig_nSV[p], "2D impact parameter (flight distance) significance of the " + s + ". SV", nanoaod::FlatTable::FloatColumn, 10);
+      djTable->addColumn<float>("DeepJet_sv_d3d_" + s, sv_d3d_nSV[p], "3D impact parameter (flight distance) value of the " + s + ". SV", nanoaod::FlatTable::FloatColumn, 10);
+      djTable->addColumn<float>("DeepJet_sv_d3dsig_" + s, sv_d3dsig_nSV[p], "3D impact parameter (flight distance) significance of the " + s + ". SV", nanoaod::FlatTable::FloatColumn, 10);
+      djTable->addColumn<float>("DeepJet_sv_costhetasvpv_" + s, sv_costhetasvpv_nSV[p], "cosine of the angle between the " + s + ". SV flight direction and the direction of the " + s + ". SV momentum", nanoaod::FlatTable::FloatColumn, 10);
+      /*
+      // anstein: only relevant if also included in the tag info, not yet, maybe in future versions of the tagger
```

Comment:
keep these, as they might be relevant later, but as a rule most comments could/should be removed